### PR TITLE
feat(plasma cannon): add plasma cannon sound

### PR DESCRIPTION
**Pull request by EikeSan**
Added plasma cannon sound effect and changed compnent factory

### CODE REVIEW

File: src/main/java/org/openRealmOfStars/player/ship/ShipComponentFactory.java [hunk 1]
```diff
@@ -2477,28 +2477,28 @@ private static ShipComponent createWeapon(final int index) {
       tmp.setEnergyRequirement(1);
     }
     if (index == COMPONENT_PLASMA_CANNON_MK2) {
-      tmp = new ShipComponent(index, "Plasma cannon Mk2", 6, 4,
```
Comment (tuomount) [inline src/main/java/org/openRealmOfStars/player/ship/ShipComponentFactory.java:2480]: Was there a reason to change production and metal cost of Plasma cannons?

**Review by tuomount — COMMENTED** [review]
You need to add credits for Plasma cannon effect and mention correct CC license for that. That is done by updating CreditsView.java. When that is being added you can run the Oros with parameter "--credits" and copy paste output to AUTHORS.md.

Sound effect itself is awesome! There is a bit long silence at begining of the sound effect. Could you remove also that?

File: src/main/java/org/openRealmOfStars/player/ship/ShipComponentFactory.java (hunk 1 above)
Comment (EikeSan) [reply src/main/java/org/openRealmOfStars/player/ship/ShipComponentFactory.java:2480]: I just thought that will increase the difficult 🙄

**Comment by EikeSan** [comment]
@tuomount added credits and rollback costs values

**Comment by tuomount** [comment]
Could you please link the original sound. No need for code change and link for example int this merge request.

**Comment by EikeSan** [comment]
Changed the sound effect

**Comment by tuomount** [comment]
Excellent. I'll accept the pull request soon.

**Pull request merged**